src/sema/namespace.rs

```diff
@@ -490,6 +507,106 @@ impl Namespace {
         }
     }
 
+    pub(super) fn resolve_error(
```

Comment:
Would be nice to have a comment for this function describing its return value, e.g. what is the usize about and in what case it will return  `Err()`

Reply:
Added comment